contracts/MainframeStake.sol

```diff
@@ -63,20 +84,36 @@ contract MainframeStake is Ownable, StakeInterface {
     requiredStake = value;
   }
 
-  function getEscrowAddress() public view returns (address) {
-    return address(escrow);
+  function refundBalances(address[] addresses) public onlyOwner {
+    for (uint256 i = 0; i < addresses.length; i++) {
```

Comment:
We could have the issue of running out of gas if there are too many addresses, no? Could we optionally provide a slice as arguments so we can run the refunds in multiple steps if needed from a client?